### PR TITLE
added Language dropdown

### PR DESCRIPTION
**Pull request by NancyAanchal**
made a language dropdown to change the language of the flashcards in future
made the header match the mockups and a lil bit

### CODE REVIEW

File: nepalingo-web/src/hooks/Langauge.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+type ValueOf<T> = T[keyof T];
+export type Language = ValueOf<typeof Languages>
+
+export type LanguageKey = keyof typeof Languages;
+
+export const Languages = {
+    "Newari": "newari",
+    "Tajpuriya": "tajpuriya",
+    "Maithili": "coming soon",
+} as const
```
Comment (christikaes) [inline nepalingo-web/src/hooks/Langauge.tsx:13]: why not just make it a string? this object type seems a bit redundant. Why is `as const` needed?

File: nepalingo-web/src/hooks/Langauge.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+type LanguageContextProps = {
+    selectedLanguage: string | null | undefined;
```
Comment (christikaes) [inline nepalingo-web/src/hooks/Langauge.tsx:17]: do we need both `null` and `undefined`?

**Review by christikaes — COMMENTED** [review]
I think this looks good! small comments and I haven't had a chance to test. If someone else can test and approve this can get merged

**Review by SansCaar — APPROVED** [review]
Everythings looks good to me.

**Pull request merged**